syftlib/src/main/java/org/openmined/syft/Job.kt

```diff
@@ -0,0 +1,45 @@
+package org.openmined.syft
+
+
+import io.reactivex.Flowable
+import io.reactivex.Single
+import io.reactivex.processors.PublishProcessor
+import kotlinx.serialization.Serializable
+import org.openmined.syft.network.NetworkMessage
+import org.openmined.syft.threading.ProcessSchedulers
+
+@Serializable
+class Job(val modelName: String, val version: String) {
+
+    /**
+     * create a worker job (throw warning if more than 1 job are created)
```

Comment:
Shouldn't this be controlled by `Syft` instead?

Reply:
Yeah, the number of workers will be controlled by syft. My bad!